init.lua

```diff
@@ -3,23 +3,27 @@
 -- Copyright © 2014-2020 4aiman, Hugo Locurcio and contributors - MIT License
 -- See `LICENSE.md` included in the source distribution for details.
 
-if minetest.settings:get_bool("health_bars") == false or
-		not minetest.settings:get_bool("enable_damage")
-then return end
+local enabled = minetest.settings:get_bool("health_bars")
+if enabled == nil then
+	enabled = minetest.settings:get_bool("enable_damage")
+end
 
 -- Localize the vector distance function for better performance,
 -- as it's called on every step
 local vector_distance = vector.distance
 
+local mt_version = minetest.get_version().string
+local mt_5 = tonumber(mt_version:sub(1, 1)) >= 5
+local mt_53 = mt_5 and tonumber(mt_version:sub(2, 2)) >= 3
```

Comment:
This is a really bad hack. Could you please instead use `minetest.global_exists` for `"player_api"`, or checking against a 5.0.0+ API function?

Reply:
```
local mt_5 = minetest.features and minetest.features.object_independent_selectionbox
local mt_53 = mt_5 and ???
```

Reply:
`minetest.features.object_independent_selectionbox` is the change that made the player models incompatible, so that would be the best choice.
Also you don't need to check against `minetest.features`. It exists for sure.